app/controllers/lessons_controller.rb

```diff
@@ -1,5 +1,18 @@
 class LessonsController < ApplicationController
+  before_action :is_admin?
   def index
     @category = Category.paginate(page: params[:page], per_page: 6 )
+    @learned = Lesson.joins("JOIN categories ON lessons.category_id = categories.id")
+                      .select("categories.*, lessons.result")
+                      .where("result = 1")
+                      .paginate(page: params[:page], per_page: 6 )
+    @not_learned = Category.joins("JOIN lessons ON lessons.category_id = categories.id")
+                         .select("*") #*全て
+                         .where("lessons.category_id = null")
+                         .paginate(page: params[:page], per_page: 6 )
```

Comment:
we should be able to see a result from this variable, you can test this SQL Query inside Workbench to see the exepected results:
`SELECT * FROM e_learning_system_development.categories LEFT OUTER JOIN e_learning_system_development.lessons ON lessons.category_id = categories.id WHERE category_id IS NULL;`
![image](https://user-images.githubusercontent.com/73863408/129877250-9212ce97-8af8-4ab4-854d-102656a67c43.png)


as you can see, all the results from lessons are null, but the ones in the categories are there.. this is how we are able to see the list of lessons that we did not take yet

with your current implementation, this is what it looks like, which is _not correct_:
![image](https://user-images.githubusercontent.com/73863408/129877473-cc0af8a0-486d-40ac-8e8a-50a0b56d98b9.png)